src/schemas/VIC/schema.js

```diff
@@ -1,13 +1,77 @@
 import originalDefinitions from '../../common/definitions';
 import schemaHelpers from '../../common/schema-helpers';
 import _ from 'lodash';
+import constants from '../../common/constants';
+
+const address = (() => {
+  const countries = constants.salesforceCountries.map(object => object.value);
+  const countriesWithStates = Object.keys(constants.salesforceStates).filter(x => _.includes(countries, x));
+  const countriesWithNoState = _.difference(countries, countriesWithStates);
+  const countryStateProperties = _.map(constants.salesforceStates, (value, key) => ({
+    properties: {
+      country: {
+        type: 'string',
+        'enum': [key]
+      },
+      state: {
+        type: 'string',
+        'enum': value.map(x => x.value)
+      },
+      postalCode: {
+        type: 'string',
+        maxLength: 10
+      }
+    }
+  }));
+  countryStateProperties.push({
+    properties: {
+      country: {
+        not: {
```

Comment:
We could probably get rid of this, right? Since the countriesWithNoState list would skip these?